app/controlplane/plugins/sdk/v1/plugin/api/translation.go

```diff
@@ -109,21 +110,51 @@ func AttachProtoToSDK(in *AttachResponse) (*sdk.AttachmentResponse, error) {
 }
 
 func MetadataSDKToProto(in *sdk.ChainloopMetadata) *ExecuteRequest_Metadata {
```

Comment:
the new plugin system needed to be extended to know how to translate the new metadata payload